src/wazuh_modules/agent_upgrade/manager/wm_agent_upgrade_commands.c

```diff
@@ -45,23 +47,25 @@ static cJSON* wm_agent_upgrade_analyze_agent(int agent_id, wm_agent_task *agent_
  * @retval WM_UPGRADE_WPK_SHA1_DOES_NOT_MATCH
  * @retval WM_UPGRADE_UNKNOWN_ERROR
  * */
-static int wm_agent_upgrade_validate_agent_task(const wm_agent_task *agent_task);
+static int wm_agent_upgrade_validate_agent_task(const wm_agent_task *agent_task, wm_manager_configs manager_configs);
 
 /**
  * Start the upgrade procedure for the agents
  * @param json_response cJSON array where the responses for each agent will be stored
  * @param task_module_request cJSON array with the agents to be upgraded
+ * @param manager_configs manager configuration parameters
  * */
-static void wm_agent_upgrade_start_upgrades(cJSON *json_response, const cJSON* task_module_request);
+static void wm_agent_upgrade_start_upgrades(cJSON *json_response, const cJSON* task_module_request, wm_manager_configs manager_configs);
 
 /**
  * Send WPK file to agent and verify SHA1
  * @param agent_task structure with the information of the agent and the WPK
+ * @param manager_configs manager configuration parameters
  * @return error code
  * @retval OS_SUCCESS on success
  * @retval OS_INVALID on errors
  * */
-static int wm_agent_upgrade_send_wpk_to_agent(const wm_agent_task *agent_task);
+static int wm_agent_upgrade_send_wpk_to_agent(const wm_agent_task *agent_task, wm_manager_configs manager_configs);
```

Comment:
Since `manager_configs` is used as read-only it could be defined as `const`. It is also better to use a pointer, to avoid creating a copy of the structure.

`const wm_manager_configs* manager_configs`